BraveRewardsUI/Settings/Ads/Details/AdsDetailsViewController.swift

```diff
@@ -137,6 +221,22 @@ extension AdsDetailsViewController: UITableViewDelegate, UITableViewDataSource {
       if adsPerHour - 1 < adsPerHourOptions.count {
         cell.accessoryLabel?.text = adsPerHourOptions[adsPerHour - 1]
       }
+    case .subdivisionTargeting:
+      cell.accessoryType = .disclosureIndicator
+
+      cell.label.text = Strings.adsSubdivisionTargeting
+
+      var adsSubdivisionTargetingCode: String
+      if state.ads.subdivisionTargetingCode == "DISABLED" {
```

Comment:
Is there no const definition of this on ads library side that can be used to avoid string-checking here and L232?

Reply:
Not at this time, however, I will raise a new issue to add an API call to detect if disabled. So recommended we add at a future date, thanks